src/System.Net.Requests/tests/FileWebRequestTest.cs

```diff
@@ -80,7 +80,7 @@ public void GetRequestResponseAfterAbort_Throws()
         [Fact]
         public void NotImplementedMembers_Throws()
         {
-            WebRequest request = WebRequest.Create("file://anything");
+            WebRequest request = WebRequest.Create("file:///anything");
```

Comment:
> Why does that have to change? I'm concerned about this being a breaking change. Are there no valid situations someone could have been using this today without the extra '/'?

I wrote the requirement more strict at first. The path could not be empty and had to start with a slash '/'. At that point, I updated the tests. I then relaxed it to, if there is a path it must start with '/'.
```
file://anything -> host=anything, path=/
file:///anything -> host=, path=/anything
```
So the tests will now also work with 'file://anything'.